x-pack/plugins/ingest_manager/server/services/agents/actions.ts

```diff
@@ -247,11 +247,16 @@ export async function getNewActionsSince(
         nodeTypes.literal.buildNode(false),
       ])
     ),
+    nodeTypes.function.buildNodeWithArgumentNodes('is', [
```

Comment:
added this condition to not fetch `POLICY_CHANGE` here, (the action was filtered in the code after) but in my opinion the earlier we filter it the better it is